specification/app/resource-manager/Microsoft.App/preview/2024-02-02-preview/examples/ContainerApps_CreateOrUpdate.json

```diff
@@ -96,7 +96,7 @@
               "enableMetrics": true,
               "javaAgent": {
                 "enabled": true,
-                "logLevelSettings": [
+                "loggerSettings": [
```

Comment:
Let's nest this array inside another property to allow space for logging settings that don't apply to specific loggers in the future:

```
"logging": {
  "loggerSettings": [
     "logger": "org.springframework.boot",
     "level": "debug"
  ]
}
```

Reply:
updated